Applications/ISIS.md

```diff
@@ -0,0 +1,74 @@
+## Introduction
+The ISIS software project is a collection of planetary image processing applications for data collected by a wide range of sensors. The source code and issue tracker are hosted on [GitHub](https://github.com/USGS-Astrogeology/ISIS3) and it's documentation is hosted on a separate [website](https://isis.astrogeology.usgs.gov/).
+
+## History & Outlook
+Originating in the 1970s, ISIS has a long history that is detailed [here](https://isis.astrogeology.usgs.gov/documents/IsisHistory/IsisHistory.html).
+
+The USGS Astrogeology Science Center (ASC) has been the owner and maintainer of ISIS. Recently, the ASC has started transitioning the ISIS software project to open source and is seeking greater community involvement. A [technical committee](https://github.com/USGS-Astrogeology/ISIS_TC) has been stood up and governance of the project is being gradually transferred from the ASC to the ISIS technical committee (ISIS TC).
+
+## Metrics
+
+Currently, the majority of code contributors are employees or past employees of the Astrogeology Science Center (ASC). In the last year, a small number of non-ASC personnel have contributed to the project. Issues and documentation have been contributed by the broader user community.
```

Comment:
I mean these as questions on process. What kind of metrics might we ask for here? What reason might we have for asking for this type of information? (I just read @victoronline PR and saw the blurb about limited mentor resources, so that is the impetus behind these comments.)

Reply:
I think any kind the submitting project thinks are relevant.  Usage metrics and/or contribution metrics, I guess.